guides/v2.1/extension-dev-guide/versioning/codebase-changes.md

```diff
@@ -150,6 +157,18 @@ Use this table to understand what changes Magento can make and which version num
 | | Index added/changed| PATCH|
 | | Foreign key added| MAJOR|
 | | Temporary tables added/removed/changed | PATCH|
-  
-  
-[private]: http://php.net/manual/en/language.oop5.visibility.php
+| **Static view files (marked with `@api`)** | JS/CSS/LESS file removed/moved | MAJOR|
+| | JS/CSS/LESS file added | PATCH|
+
+### PHP Classes Intended for Extension
```

Comment:
We should note what's different about versioning for these classes.  Also, having some annotation in the code for such classes would be very useful (both for extension development and semver tooling).